editor.planx.uk/src/@planx/components/FindProperty/Public/Map.tsx

```diff
@@ -0,0 +1,165 @@
+}
+
+export const DescriptionInput = styled(Box)(({ theme }) => ({
+  paddingTop: theme.spacing(1),
+  paddingBottom: theme.spacing(2),
+}));
+
+export default function PlotNewAddress(props: PlotNewAddressProps): FCReturn {
+  const formik = useFormik<ProposedAddressInputs>({
+    initialValues: {
+      siteDescription: props.initialProposedAddress?.title || "",
+    },
+    onSubmit: (values) => {
+      if (proposedAddress) {
+        props.setAddress({
+          ...proposedAddress,
+          title: formik.values.siteDescription,
+        });
+      }
+    },
+    validateOnBlur: false,
+    validateOnChange: false,
+    validationSchema: userDataSchema,
+  });
+
+  const [proposedAddress, setProposedAddress] = useState<
+    SiteAddress | undefined
+  >(props.initialProposedAddress ?? undefined);
+
+  const environment = useStore((state) => state.previewEnvironment);
+
+  useEffect(() => {
+    const geojsonChangeHandler = ({ detail: geojson }: any) => {
+      if (geojson["EPSG:3857"]?.features && geojson["EPSG:27700"]?.features) {
+        // only a single point can be plotted, so get first feature in geojson "FeatureCollection" per projection
+        const [longitude, latitude] =
+          geojson["EPSG:3857"].features[0]?.geometry?.coordinates;
+        const [x, y] = geojson["EPSG:27700"].features[0]?.geometry?.coordinates;
+
+        setProposedAddress({
+          latitude,
+          longitude,
+          x,
+          y,
+          title: formik.values.siteDescription || "",
+          source: "proposed",
+        });
+      } else {
+        // triggered if a user "clears" their point on the map
+        setProposedAddress(undefined);
+      }
+    };
+
+    const map: any = document.getElementById("plot-new-address-map");
+    map?.addEventListener("geojsonChange", geojsonChangeHandler);
+
+    return function cleanup() {
+      map?.removeEventListener("geojsonChange", geojsonChangeHandler);
+    };
+  }, [setProposedAddress]);
+
+  return (
+    <Card
+      handleSubmit={formik.handleSubmit}
+      isValid={
+        Boolean(proposedAddress) && Boolean(formik.values.siteDescription)
+      }
+    >
+      <QuestionHeader
+        title={props.title || DEFAULT_NEW_ADDRESS_TITLE}
+        description={props.description || ""}
+      />
+      <MapContainer environment={environment} size="large">
+        <p style={visuallyHidden}>
+          An interactive map centred on the local authority district, showing
+          the Ordnance Survey basemap. Click to place a point representing your
+          proposed site location.
+        </p>
+        {/* @ts-ignore */}
+        <my-map
+          id="plot-new-address-map"
+          data-testid="map-web-component"
+          zoom={14}
+          drawMode
+          drawType="Point"
+          resetControlImage="trash"
+          showScale
+          showNorthArrow
+          osVectorTilesApiKey={process.env.REACT_APP_ORDNANCE_SURVEY_KEY}
```

Comment:
This should be pointing to the proxy - I'm actually not sure this is displaying OS tiles as this env var shouldn't be set. 

Looking into it now...!

Reply:
updated to point to proxy now - working as expected! think this was just a rebase problem on my side, this PRs been lingering too long :upside_down_face: